package.json

```diff
@@ -68,6 +68,12 @@
       ]
     ]
   },
+  "nodemonConfig": {
+    "verbose": true,
+    "ignore": ["assets", "public"],
+    "exec": "bin/www",
+    "ext": "js,json,njk"
```

Comment:
I know this is merged now, but `.njk` files should have already been refreshing on change (eg. no need for the app to reload)?

Reply:
Hmm, interesting. Will double check.